apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/VideoView.tsx

```diff
@@ -48,7 +48,7 @@ export function VideoView(): ReactElement {
         }}
       >
         <Typography variant="h4">
-          {t('titleEdit', { value: videoTitle })}
+          {t('videoTitle', { value: videoTitle })}
```

Comment:
no need for interpolation anymore
```suggestion
          {videoTitle}
```